lib/msal-core/src/utils/ErrorUtils.ts

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { AuthCache } from "../cache/AuthCache";
+import { AuthError } from "../error/AuthError";
+/**
+ * @hidden
+ */
+export class ErrorUtils {
+
+    static errorHandler(storage: AuthCache, errorFunc: AuthError): AuthError {
```

Comment:
Why is it called `errorFunc`? It's an actual error, correct?

Reply:
And I would consider being more explicit with the name: `throwErrorAndClearTempCache`

Reply:
Renamed and renamed

Reply:
Function no longer needed. Please check the latest code.